src/Configuration.php

```diff
@@ -261,11 +261,24 @@ public function getFilters()
      */
     public function setProjectRoot($projectRoot)
     {
-        $this->projectRootRegex = $projectRoot ? '/^'.preg_quote($projectRoot, '/').'[\\/]?/i' : null;
+        $this->setProjectRootRegex('/^'.preg_quote($projectRoot, '/').'[\\/]?/i');
```

Comment:
This should still check for and allow a null value to be set without throwing an exception in setProjectRootRegex.

Reply:
Added some checks to enable the `stripPath` & `projectRoot` to be null.  As we already check for truthiness in the `isInProject` and `getStrippedFilePath` methods this will cover these cases.